Source/JavaScriptCore/inspector/remote/RemoteInspector.cpp

```diff
@@ -56,10 +56,8 @@ TargetID RemoteInspector::nextAvailableTargetIdentifier()
     return nextValidTargetIdentifier;
 }
 
-void RemoteInspector::registerTarget(RemoteControllableTarget* target)
+void RemoteInspector::registerTarget(Ref<RemoteControllableTarget> target)
```

Comment:
This should keep passing a raw pointer or C++ reference. It's the call site's job to project the object it is passing in argument.